### PR TITLE
secsan: Add tags field to event data on new vulnerability notifications (PROJQUAY-5681)

### PR DESCRIPTION
**Pull request by Marcusk19**
Add tag field to event data for vulnerability notifications on new image pushes.

Missing tag field caused an exception for slack notifications, should allow slack notifications to fire for new images.

Screenshot 2023-06-29 at 11 49 55 AM

### CODE REVIEW

File: data/secscan_model/secscan_v4_model.py [hunk 1]
```diff
@@ -440,10 +442,15 @@ def should_skip_indexing(manifest_candidate):
                             import notifications
 
                             keys = list(found_vulnerabilities)
+                            tag_names = list(
+                                registry_model.tag_names_for_manifest(manifest, TAG_LIMIT)
+                            )
+                            print("TAG_NAMES: ", tag_names)
```
Comment (kleesc) [inline data/secscan_model/secscan_v4_model.py:448]: Forgot to remove this print statement

File: data/secscan_model/secscan_v4_model.py [hunk 2]
```diff
@@ -56,6 +57,7 @@
 
 
 DEFAULT_SECURITY_SCANNER_V4_REINDEX_THRESHOLD = 86400  # 1 day
+TAG_LIMIT = 100
```
Comment (hdonnay) [inline data/secscan_model/secscan_v4_model.py:60]: Why the limit? What subset is chosen when it's exceeded?

Comment (Marcusk19) [reply data/secscan_model/secscan_v4_model.py:60]: From what I understand the limit is for retrieving the tag names that point to that manifest up to that limit. So at max there will only be 100 tags attached to the event data for the notification.

Comment (hdonnay) [reply data/secscan_model/secscan_v4_model.py:60]: Right, so how is that 100 chosen?
The limit will have to be documented somewhere, also.

Alternatively, is it possible to page through all the tags and emit multiple events if needed?

Comment (Marcusk19) [reply data/secscan_model/secscan_v4_model.py:60]: The security notification worker already sets it as 100 which is why I chose that number. It's also possible to set it as None for no limit. 
Is it better to just go through all tag names and not limit it to 100?

Comment (hdonnay) [reply data/secscan_model/secscan_v4_model.py:60]: Maybe? It could make the event object very large, which is why I suggested paging.

**Pull request closed without merging**